index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Portfolio Mobile version</title>
+    <meta
+      name="description"
+      content="Portfolio mobile version for my projects and learning in bootcamp"
+    />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <link
+      rel="alternate"
+      type="application/atom+xml"
+      title="Portfolio Mobile version"
+      href="/portfolio/"
+    />
+    <link
+      rel="search"
+      type="application/opensearchdescription+xml"
+      title="Portfolio Mobile version"
+      href="opensearch.xml"
+    />
+    <link rel="shortcut icon" href="/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" type="text/css" href="core.css" media="screen" />
+    <link rel="stylesheet" type="text/css" href="print.css" media="print" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/3.0.3/normalize.min.css"
+      integrity="sha512-Ojqt7YpXqYM6//AdMhErV3ot38rYgGF5QLJEwx7zhesjL9VqfhWiRz/dWK22hsn96RNz0CLl85+pg1P0BmfgVQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+    rel="stylesheet"
+  />
+  </head>
+  <body>
+    <section class="header">
+        <nav class="title_lo">
+          <a class="logo" href="#" target="_blank" rel="noopener noreferrer"
+            >My Logo</a
+          >
+          <i class="fas fa-bars"></i>
+        </nav>
+      </header>
+      </section>
+      <section class="content-name">
```

Comment:
- [ ] This is so great the efforts you made to have the same design as the one in Figma, but I can see that there are some areas where you did not follow the Figma template if you take a look at the mobile version of your project the margins are not matching the Figma Template. :handshake: 
![image](https://user-images.githubusercontent.com/28109626/148310230-479e852f-6454-4345-b6ae-082255294acb.png)

You can see your project in mobile version, using the developer tools of your web browser.